catalog/security/banner-marking/src/test/groovy/org/codice/alliance/security/banner/marking/BannerCommonMarkingExtractorSpec.groovy

```diff
@@ -319,6 +312,114 @@ class BannerCommonMarkingExtractorSpec extends Specification {
         '//COSMIC RESTRICTED//BOHEMIA' | true            | null
     }
 
+    def 'test security conflict classification'() {
+        when:
+        initBannerMarkings("TOP SECRET")
+        metacard.setAttribute(new AttributeImpl(Security.CLASSIFICATION, "CONFLICTING VALUE"))
```

Comment:
✏️ I think your tests would be more definitive and less prone to being broken in the future if you used real values. So instead of CONFLICTING VALUE here, use a real classification value such as `SECRET` or `S`.